react/training/src/app/screens/Game/index.js

```diff
@@ -1,16 +1,18 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { handleClick, jumpTo } from '../../../redux/game/actions/action';
```

Comment:
You have to import all actions like this: `import { gameActions } from '../../../redux/game/actions';` with only one name and then you can call it `gameActions.xxx or gameActions.yyy`